services/madoc-ts/src/frontend/site/blocks/ProjectActions.tsx

```diff
@@ -41,8 +41,8 @@ export const ProjectActions: React.FC<{
             as={Link}
             to={
               options.reviewerDashboard
-                ? createLink({ projectId: project.id, subRoute: 'reviews' })
-                : createLink({ projectId: project.id, subRoute: 'tasks', query: { type: 'crowdsourcing-review' } })
+                ? createLink({ projectId: project.id, subRoute: 'tasks', query: { type: 'crowdsourcing-review' } })
+                : createLink({ projectId: project.id, subRoute: 'reviews' })
```

Comment:
This flip will make users who have previously opted in go to the old one - We could always direct to the review dashboard instead - I saw you added a link to the task list if people need to get back to that view